config/main.go

```diff
@@ -11,7 +11,7 @@ type Config struct {
 	ZoneName      string
 	SRVRecordName string
 	TTL           uint16
-	SRVRecord     net.SRV
+	SRVRecord     *net.SRV
```

Comment:
After this PR, the `Config` struct isn't passed down anywhere anymore, so its not that much of a singleton anymore… In theory, it could all be locally scoped variables.

I could indeed keep this as-is, pass a pointer to `checker.Run` in https://github.com/wireapp/srv-announcer/pull/8/files#diff-7ddfb3e035b42cd70649cc33393fe32cR139 wouldn't need to initialize in https://github.com/wireapp/srv-announcer/pull/8/files#diff-7ddfb3e035b42cd70649cc33393fe32cR25 - no strong preference here.